application/models/User.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+class User {
+
+  /**
+   * Stores user first name.
+   *
+   * @var string
```

Comment:
It should be of the form `@var string $firstName`

Reply:
The variable name does not need to write to `@var string $firstName` in this format you can write (i.e: `@var string`) this also.